package/MDAnalysis/coordinates/GSD.py

```diff
@@ -131,3 +135,4 @@ def _read_frame(self, frame):
     def _read_next_timestep(self) :
         """read next frame in trajectory"""
         return self._read_frame(self._frame + 1)
+
```

Comment:
PEP8 don't need a blank line on the last line

Reply:
@IAlibay do we have a PEP8 checker running? If not, we should add one to the linter.

@lilyminium does the user guide talk about PEP8-checking? 

I'm just asking I find myself writing a lot of "PEP8" comments on PRs.

Reply:
I'm not sure we do - I think there's already an ongoing discussion in #2450

Reply:
> does the user guide talk about PEP8-checking?

Yes, the [user guide](https://www.mdanalysis.org/UserGuide/contributing_code.html#code-formatting-in-python) highlights important points from PEP8, mentions tools like flake8 for linting, and autopep8/yapf for autoformatting. It doesn't mention stuff like spaces after commas and around comparison operators, or blank lines, though.